packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -53,15 +65,20 @@ $sidebar-width-desktop: 324px;
 	.components-modal__content {
 		overflow-y: scroll;
 		-webkit-overflow-scrolling: touch;
-		// prevents a 1px strip of content being visible after scrolling
-		margin-top: -1px;
+		margin-top: 60px;
 		padding: 0 16px 16px;
+
+		&::before {
+			margin-bottom: 0;
```

Comment:
This is for correcting where the scroll bar shows, right? Let's add a comment about that!

Reply:
No, it's just functioning like a clearfix in the core component. Added a comment and just `display: none` instead.